README.md

```diff
@@ -199,24 +201,248 @@ Runs all test suites, will be run automatically by GitHub to ensure that all cod
 
 #### `yarn test:build`
 
 Runs tests to validate that the build is valid and up to date.
 
 #### `yarn lint`
 
-Checks that all files (code and songs) follows style rules.
+Checks that all files (code and songs) follow style rules.
 
 #### `yarn format`
 
 Formats all files (code and songs) to follow style rules.
 
 ## Getting help
 
-This might all sound like alot so if there's ever anything you're wondering about you can always ask for help! We want anyone to be able to contribute, even if you've never contributed to an open-source repository before or never used git. So if there's anything you don't understand or need clarification on feel free to create an issue on this repository marking it as a question. And of course there's always the good old fashioned asking someone you know for help.
+This might all sound like a lot so if there's ever anything you're wondering about you can always ask for help! We want anyone to be able to contribute, even if you've never contributed to an open-source repository before or never used Git. So if there's anything you don't understand or need clarification on feel free to create an issue on this repository marking it as a question. And of course, there's always the good old fashioned asking someone you know for help.
 
 ## Background
 
 The songs in this IN-chapter song list are originally from Strängteoretiquerna's revised songbook (2009-2015) with additions, deletions and revisions made in the [in-sangbok repository](https://github.com/wsv-accidis/in-sangbok) between the years 2016 to 2022.
+
+# Sånglista för IN-Sektionen
+
+Detta är IN-Sektionens sånglista. Sångerna är sparade som Markdown filer i [`/songs`](/songs) mappen och genererade till sånglistor i olika format.
+
+Du kan också hitta information hur du bidrar till detta repository, antingen genom att lägga till/ändra sånger eller genom att bidra till koden.
+
+## Innehållsförteckning <!-- omit in toc -->
+
+- [Bidra (allmänt)](#bidra-allmänt)
+  - [Snabbguide](#snabbguide)
+- [Bidra (sånger)](#bidra-sånger)
+  - [Format](#format)
+  - [Skapa](#skapa)
+  - [Ändra](#ändra)
+  - [Validera](#validera)
+  - [Kompilera](#kompilera)
+- [Bidra (koden)](#bidra-koden)
+- [Script](#script)
+  - [Sångscript](#sångskript)
+  - [Projektscript](#projektscript)
+- [Få hjälp](#få-hjälp)
+- [Bakgrund](#bakgrund)
+
+## Bidra (allmänt)
+
+Vill du hjälpa till att bidra? Grymt! Här hittar du allt du behöver veta och hur du ska göra det på rätt sätt.
+
+Först och främst, all kod i detta projektet är skrivet i [TypeScript](http://typescriptlang.org/), så för att kunna använda det behöver du [Node.js](https://nodejs.dev/) installerat och (valfritt men rekommenderat) [Yarn (classic)](https://classic.yarnpkg.com/) installerat. Om du inte vill använda Yarn kan du använda NPM istället, men all dokumentation är skriven med Yarn i åtanke.
+
+Detta behövs för att kunna kompilera nya sånger, testa och linting.
+
+### Snabbguide
+
+#### **1. Grena och klona**
```

Comment:
I think it's better to stick with the English words for various Git commands.

```suggestion
#### **1. Fork:a och Clone:a**
```

Reply:
Resolved with fixup